dev-db/tarantool/tarantool-9999.ebuild

```diff
@@ -51,6 +51,8 @@ LICENSE="BSD-2"
 BDEPEND=">=dev-util/cmake-2.6"
 
 RDEPEND="
+	acct-user/tarantool
+	acct-group/tarantool
```

Comment:
We have the following code in `src_install`:

```shell
# Log directory
keepdir /var/log/tarantool
fowners "${TARANTOOL_USER}:${TARANTOOL_GROUP}" /var/log/tarantool
```

A quote from `acct-user.eclass`:

```shell
# Then you add appropriate dependencies to your package.  Note that
# the build system might need to resolve names, too.  The dependency
# type(s) should be: BDEPEND if the user must be resolvable at build
# time (e.g. 'fowners' uses it in src_install), IDEPEND if it must be
# resolvable at install time (e.g. 'fowners' uses it in pkg_preinst),
# and RDEPEND in every case.
```

It seems, it should be in `BDEPEND` in our case.

Reply:
Fixed